pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[tool.mypy]
+check_untyped_defs = true
+disallow_any_generics = true
+disallow_incomplete_defs = true
+disallow_untyped_calls = false
+disallow_untyped_defs = true
+no_implicit_optional = true
+show_error_codes = true
+strict_equality = true
+strict_optional = true
+warn_redundant_casts = true
+warn_unreachable = true
+warn_unused_configs = true
+warn_unused_ignores = true
+
+[[tool.mypy.overrides]]
+module = [
+    "androguard.*",
+    "uplink",
+    "requests.*",
+    "authlib.*",
+    "pytest",
+    "pyaxmlparser.*"
+]
+ignore_missing_imports = true
+
+[tool.ruff]
+select = ["B", "C", "D", "E", "F", "I", "N", "S", "U", "W"]
+ignore = ["E501", "D212", "D103", "D107"]
+target-version = "py310"
+src = ["resources"]
+
+[tool.ruff.per-file-ignores]
```

Comment:
@sanzoghenzo Could you tell me, if for the Hyperion Capture add-on any files need to be listed here?

Reply:
Usually I  start without/with a blank section, then I run the linters (`pipenv run pre-commit` the first time, then `pipenv run lint`) and add files and rules codes if they are not really/easily fixable

Reply:
I got the idea and either fixed small things or excluded for now.